site/source/tutorials/008_animations/demo/finished/zombies/README.md

```diff
@@ -0,0 +1,25 @@
+# animationTutorial
+
+This project was generated with the [Dojo CLI](https://github.com/dojo/cli) & [Dojo CLI create app command](https://github.com/dojo/cli-create-app).
```

Comment:
Might be worth a link to https://dojo.io/tutorials/1050_development_environment/ to give some context?